llmfoundry/utils/builders.py

```diff
@@ -159,6 +161,7 @@ def build_callback(
     name: str,
     kwargs: Union[DictConfig, Dict[str, Any]],
     config: Any = None,
+    train_loader: Union[None, DataLoader, Iterable] = None,
```

Comment:
I don't like this but I can't think of a cleaner way. @dakinggg  thoughts?

Reply:
addressed with `BEFORE_LOAD` event.

Reply:
remove